src/app/dev/DevToys.Tools/Tools/Sample/SampleToolFactory.cs

```diff
@@ -42,12 +41,13 @@ public ValueTask<int> InvokeAsync(ILogger logger, CancellationToken cancellation
     IconGlyph = "\uE670",
     GroupName = "Encoders / Decoders",
     ResourceManagerAssemblyIdentifier = nameof(DevToysToolsResourceManagerAssemblyIdentifier),
-    ResourceManagerBaseName = "DevToys.Tools.Sample.Sample",
+    ResourceManagerBaseName = "DevToys.Tools.Tools.Sample.Sample",
     ShortDisplayTitleResourceName = nameof(Sample.CommandDescription),
     LongDisplayTitleResourceName = nameof(Sample.LongTitle),
     DescriptionResourceName = nameof(Sample.Description),
     AccessibleNameResourceName = nameof(Sample.CommandDescription),
     SearchKeywordsResourceName = nameof(Sample.CommandDescription))]
+[AcceptedDataTypeName(PredefinedCommonDataTypeNames.Json)]
```

Comment:
@btiteux , here is how a tool can tell that it accepts a JSON as an input from Smart Detection